src/scripts/navbar.js

```diff
@@ -0,0 +1,61 @@
+window.onscroll = function () {
+  scrollFunction();
+};
+
+let navbar = document.getElementById("navbar");
+const hamburgerMenu = document.getElementById("menuIcon");
+const overlay = document.querySelector(".overlay");
+const navbarRight = document.getElementById("navbar-right");
+const dropdown = document.querySelectorAll(".dropdown");
+
+function scrollFunction() {
+  if (document.body.scrollTop > 5 || document.documentElement.scrollTop > 5) {
+    navbar.style.height = "58px";
+    document.getElementById("navbar-right").style.lineHeight = "58px";
+    document.getElementById("navbar-right").style.top = "58px";
+    document.getElementById("navbar-left").style.lineHeight = "58px";
```

Comment:
Setting these styles through javascript should only be done if you are setting dynamic styles, otherwise you should add these style changes to a specific css class and only add or remove that class:

```
navbarRight.classList.add(scrolledClass);
```

Also, you've already defined those elements in variables on the top of the file so you should use those variables instead of querying for the element every time